### PR TITLE
licenseplist 3.22.5 (new formula)

### PR DESCRIPTION
**Pull request by umireon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
```
  error: link command failed with exit code 1 (use -v to see invocation)
  Undefined symbols for architecture arm64:
    "_swift_async_extendedFramePointerFlags", referenced from:
        _$s14ArgumentParser20AsyncParsableCommandPAAE4mainyyYaFZ in AsyncParsableCommand.swift.o
        _$s14ArgumentParser20AsyncParsableCommandPAAE4mainyyYaFZTQ0_ in AsyncParsableCommand.swift.o
        _$s14ArgumentParser17AsyncMainProtocolPAAE4mainyyYaFZ in AsyncParsableCommand.swift.o
        _$s14ArgumentParser17AsyncMainProtocolPAAE4mainyyYaFZTQ0_ in AsyncParsableCommand.swift.o
        __swift_async_extendedFramePointerFlagsUser in AsyncParsableCommand.swift.o
       (maybe you meant: __swift_async_extendedFramePointerFlagsUser)
    "_swift_task_alloc", referenced from:
        _$s14ArgumentParser20AsyncParsableCommandPAAE4mainyyYaFZ in AsyncParsableCommand.swift.o
        _$s14ArgumentParser17AsyncMainProtocolPAAE4mainyyYaFZ in AsyncParsableCommand.swift.o
    "_swift_task_dealloc", referenced from:
        _$s14ArgumentParser20AsyncParsableCommandPAAE4mainyyYaFZTQ0_ in AsyncParsableCommand.swift.o
        _$s14ArgumentParser17AsyncMainProtocolPAAE4mainyyYaFZTQ0_ in AsyncParsableCommand.swift.o
  ld: symbol(s) not found for architecture arm64
```

**Comment by umireon** [comment]
Asking upstream: https://github.com/mono0926/LicensePlist/issues/197

**Comment by umireon** [comment]
The patch proposed on upstream https://github.com/mono0926/LicensePlist/pull/198

**Comment by umireon** [comment]
@carlocab All the problems I acknowledged were solved!

File: Formula/licenseplist.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Licenseplist < Formula
+  desc "License list generator of all your dependencies for iOS applications"
+  homepage "https://www.slideshare.net/mono0926/licenseplist-a-license-list-generator-of-all-your-dependencies-for-ios-applications"
+  url "https://github.com/mono0926/LicensePlist/archive/refs/tags/3.22.5.tar.gz"
+  sha256 "568c3bb40784ee59d5ebf12a6249d95f2eda2f2aabd43ad5be57e1c5db5f7b08"
+  license "MIT"
+
+  depends_on xcode: ["13.0", :build]
+  depends_on :macos
```
Comment (carlocab) [inline Formula/licenseplist.rb:9]: Does it actually need macOS or does it just not build with the `swift` that we ship?

**Review by carlocab — DISMISSED** [review]
Thanks!

File: Formula/licenseplist.rb (hunk 1 above)
Comment (umireon) [reply Formula/licenseplist.rb:9]: The current implementation of LicensePlist depends on a library named APIKit which is not compatible with Linux's Swift 5.5 due to the `FoundationNetworking` issue.

I don't think the function of LicensePlist is specific to macOS and this restriction can be eventually removed.

**Comment by umireon** [comment]
Version changed so I fixed the commit message

**Review by carlocab — APPROVED** [review]

**Review by alebcay — APPROVED** [review]

**Pull request closed without merging**